sys/conf/Makefile.mips

```diff
@@ -46,9 +50,14 @@ ARCH_FLAGS?=-march=mips64
 .else
 ARCH_FLAGS?=-march=mips32
 .endif
+
 ARCH_FLAGS+=-mabi=${MIPS_ABI}
-EXTRA_FLAGS=-fno-pic -mno-abicalls -G0 -DKERNLOADADDR=${KERNLOADADDR}
-EXTRA_FLAGS+=-${MIPS_ENDIAN}
+.if ${MACHINE_ABI:Mpurecap}
+EXTRA_FLAGS=-fpic -mcpu=cheri128 -mxcaptable
```

Comment:
I am surprised we need `-mcpu=cheri128` given it's not there for a hybrid kernel. Modules also don't use this file (right?) so are clearly getting it from the right place. Is this just outdated?

Reply:
I think this comes from a very old configuration file. I tested the builds again, turns out that `-mxcaptable` is also not required anymore.